index.js

```diff
@@ -133,9 +133,9 @@ async function send (ctx, path, opts = {}) {
     }
     ctx.set('Cache-Control', directives.join(','))
   }
-  ctx.type = type(path)
   ctx.body = fs.createReadStream(path)
-
+  ctx.type = ctx.res.type ? ctx.res.type : type(path);
```

Comment:
And for this case, it's better to check `no extension`.

If I `set type & body` before this module, and reset body here,
the response type exists, so, we just keep it.
The response type not consistent with response body :)